docs/shared/ioda.yaml

```
@@ -0,0 +1,25 @@
+ioda:
+  configuration_file:
+    base_file: path/to/config.yaml
+    update_values:
+      baz: qux
+  execution:
+    batchargs:
+      nodes: 1
+      stdout: path/to/runscript.out
```

```suggestion
      stdout: /path/to/runscript.out
```
This could also be updated, for consistency with other files, in `jedi.yaml`.